client/src/components/Disclaimer.js

```diff
@@ -1,29 +1,45 @@
-import React, {Component} from 'react';
+import React, { Component } from 'react';
 import { ReactComponent as Logo } from '../img/Logo_CORONATRACKER_Logo.svg';
 import { ReactComponent as TextLogo } from '../img/Logo_CORONATRACKER_Text_Logo.svg';
 import Button from 'react-bootstrap/Button';
 
 class Disclaimer extends Component {
+  constructor() {
+    super();
+    this.state = {
+      showPopUp: true,
+    };
+  }
 
   render() {
     return (
       <div>
-        <Logo className="LoginLogo" /> <TextLogo className="TextLogo" />
-        <p>
-          The CoronaTracker is designed to help you navigate through the pandemic with accurate information, charting your wellbeing, and tracking your health. 
-          The CoronaTracker is not intended to be used or viewed as diagnosis or treatment of disease or other symptoms, including but not limited to COVID-19.
-        </p>
-        <p>
-          This application was made by the community for the community. 
-          This application is made for you! You are in control of how you would like to contribute your data for public health and research.
-        </p>
-        <div>
-          <Button className="theme-Palette-white-text">I agree</Button>
-          <Button className="theme-Palette-white-text theme-Palette-red-bg">I don't agree</Button>
-        </div>
+        {this.state.showPopUp ? (
+          <div className="DisclaimerPopUp">
+            <div className="DisclaimerPopUpInner">
+              <Logo className="LoginLogo" className="PopUpLogo" />
+              <TextLogo className="TextLogo" className="PopUpLogo" />
+              <p>
+                The CoronaTracker is designed to help you navigate through the pandemic with accurate information,
+                charting your wellbeing, and tracking your health. The CoronaTracker is not intended to be used or
+                viewed as diagnosis or treatment of disease or other symptoms, including but not limited to COVID-19.
+              </p>
+              <p>
+                This application was made by the community for the community. This application is made for you! You are
+                in control of how you would like to contribute your data for public health and research.
+              </p>
+              <div>
+                <Button className="theme-Palette-white-text" onClick={() => this.setState({ showPopUp: false })}>
+                  I agree
+                </Button>
```

Comment:
Here, instead of setting the state within this component, we would set the state with Redux, and `DiagnosticContainer` would re-render and not even load this at all.